### PR TITLE
Add cloudbuild.yaml for docker image.

### PR DESCRIPTION
**Pull request by nmousavi**
Tested:
  manual run

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Copyright 2017 Google Inc.
```
Comment (arostamianfar) [inline cloudbuild.yaml]: 2018 :)

File: cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#
+# Redistribution and use in source and binary forms, with or without
```
Comment (arostamianfar) [inline cloudbuild.yaml]: Please use Apache 2.0 license (you can copy from vcf_to_bq.py)

File: cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# Builds docker image for gcp-variant-transforms:
+# Run using:
+# $ gcloud container builds submit --config cloudbuild.yaml --timeout 3h .
```
Comment (arostamianfar) [inline cloudbuild.yaml]: nit: please use 1h as timeout (3h is too long for this build and it usually indicates that something went wrong).

File: cloudbuild.yaml (hunk 1 above)
Comment (nmousavi) [reply cloudbuild.yaml]: Done!

File: cloudbuild.yaml (hunk 2 above)
Comment (nmousavi) [reply cloudbuild.yaml]: Done!

File: cloudbuild.yaml (hunk 3 above)
Comment (nmousavi) [reply cloudbuild.yaml]: Done!

**Pull request merged**